metadata-io/src/main/java/com/linkedin/metadata/entity/EntityService.java

```diff
@@ -442,10 +442,13 @@ protected RecordTemplate sendEventForUpdateAspectResult(@Nonnull final Urn urn,
       produceMCLTimer.stop();
 
       // For legacy reasons, keep producing to the MAE event stream without blocking ingest
-      Timer.Context produceMAETimer = MetricUtils.timer(this.getClass(), "produceMAE").time();
-      produceMetadataAuditEvent(urn, aspectName, oldValue, updatedValue, result.getOldSystemMetadata(),
-          result.getNewSystemMetadata(), MetadataAuditOperation.UPDATE);
-      produceMAETimer.stop();
+      try {
+        Timer.Context produceMAETimer = MetricUtils.timer(this.getClass(), "produceMAE").time();
+        produceMetadataAuditEvent(urn, aspectName, oldValue, updatedValue, result.getOldSystemMetadata(), result.getNewSystemMetadata(), MetadataAuditOperation.UPDATE);
+        produceMAETimer.stop();
+      } catch (Exception e) {
+        log.warn("Unable to produce legacy MAE, entity may not have legacy Snapshot schema.", e);
```

Comment:
could this result in a spammy log?

Reply:
Well, up until now it would result in a full stacktrace as this could cause ingestion to stop here. This case should only happen for:
1) A real error connecting to Kafka, we still want to mute the actual error if it's only on this topic, but should still probably warn people it's happening

2) Entity defined exclusively in entity-registry.yml with no Snapshot schema, this would either be Telemetry (happens exactly once) or a custom model in which case people might want to be aware of this behavior.

I think a warn here makes sense, but I'm okay with backpedaling it.